docs/source/user/files.md

```diff
@@ -1,32 +1,33 @@
 
 # Working with files
 
-## Overview
+## Opening files
 
-The File Browser and File menu allow you to work with files and directories on
-your system. This includes opening, creating, deleting, renaming, downloading,
-copying and sharing files and directories.
+The file browser and File menu allow you to work with
```

Comment:
@ellisonbg I thought we were hoping to emphasize proper names with capitalization in order to encourage the naming of these portions of the UI?